examples/README.adoc

```diff
@@ -48,7 +48,7 @@ The following examples are provided:
 	the device's PIN is changed from [oldpin] to <pin>.
 
 - cred [-t es256|es384|rs256|eddsa] [-k pubkey] [-ei cred_id] [-P pin]
-       [-T seconds] [-b blobkey] [-hruv] <device>
+       [-T seconds] [-b blobkey] [-hruv] [-c cred_protect_policy] <device>
```

Comment:
Would it make sense to rename the argument to `cred_protect` and use [the same notation adopted in fido2-cred](https://developers.yubico.com/libfido2/Manuals/fido2-cred.html)?